xrpl/account/main.py

```diff
@@ -1,14 +1,14 @@
 """High-level methods to obtain information about accounts."""
 
-from typing import Any, Dict, Union, cast
+import asyncio
+from typing import Dict, Union
 
-from xrpl.clients import Client, XRPLRequestFailureException
-from xrpl.core.addresscodec import is_valid_xaddress, xaddress_to_classic_address
-from xrpl.models.requests import AccountInfo
+from xrpl.asyncio.account import main
+from xrpl.clients.sync_client import SyncClient
 from xrpl.models.response import Response
 
 
-def does_account_exist(address: str, client: Client) -> bool:
+def does_account_exist(address: str, client: SyncClient) -> bool:
```

Comment:
Will these methods not work with an async client?

Reply:
no - you can't have an event loop in another event loop